### PR TITLE
Add trollsift recipe

### PR DESCRIPTION
**Pull request by djhoese**
Similar to #5883 this package is needed as a satpy dependency.

### CODE REVIEW

File: recipes/trollsift/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+package:
+  name: '{{ name|lower }}'
+  version: '{{ version }}'
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
```
Comment (mariusvniekerk) [inline recipes/trollsift/meta.yaml]: Can this be noarch?

Comment (djhoese) [reply recipes/trollsift/meta.yaml]: Done. Thanks for the suggestion. I should have thought of that.

File: recipes/trollsift/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+  license_file: ''
+  summary: String parser/formatter for PyTroll packages
+  description: Filtering modules used in PyTroll packages
+  doc_url: http://trollsift.readthedocs.io
+  dev_url: https://github.com/pytroll/trollsift
+
+extra:
+  recipe-maintainers:
+    - djhoese
+    - mraspaud
```
Comment (mariusvniekerk) [inline recipes/trollsift/meta.yaml]: @mraspaud Are you okay with being a maintainer here?

File: recipes/trollsift/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+    - adybbroe
```
Comment (mariusvniekerk) [inline recipes/trollsift/meta.yaml]: @adybbroe are you okay with being a maintainer here?

File: recipes/trollsift/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+    - pnuu
```
Comment (mariusvniekerk) [inline recipes/trollsift/meta.yaml]: @pnuu are you okay with being a maintainer on the conda forge feedstock?

File: recipes/trollsift/meta.yaml (hunk 2 above)
Comment (djhoese) [reply recipes/trollsift/meta.yaml]: @mariusvniekerk If it makes a difference these other maintainers are all core developers on satpy and all pytroll packages and are members of the pytroll organization (https://github.com/pytroll). They are also already maintainers on the `trollimage` and `aggdraw` packages which were already merged. Although I do realize they weren't publicly or explicitly asked to be maintainers on github so understand if it is preferred.

Comment (mraspaud) [reply recipes/trollsift/meta.yaml]: Yes, I'm okay with it

File: recipes/trollsift/meta.yaml (hunk 4 above)
Comment (pnuu) [reply recipes/trollsift/meta.yaml]: Yes, I'm okay with it.

File: recipes/trollsift/meta.yaml (hunk 3 above)
Comment (adybbroe) [reply recipes/trollsift/meta.yaml]: Yes, that is fine!

**Comment by djhoese** [comment]
@mariusvniekerk That's all the maintainers now.

File: recipes/trollsift/meta.yaml [hunk 5]
```diff
@@ -0,0 +1,46 @@
...
+
+test:
+  imports:
+    - trollsift
+
+about:
+  home: https://github.com/pytroll/trollsift
+  license: GPLv3+
+  license_family: GPL3
+  license_file: ''
```
Comment (jakirkham) [inline recipes/trollsift/meta.yaml]: Have raised issue ( https://github.com/pytroll/trollsift/issues/3 ) to request the license file be packaged in the future.

**Pull request merged**